project_forecast_line_deadline/models/project_task.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Therp BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    # Make forecast end date computable and stored
```

Comment:
Computable, or just plainly computed?

Reply:
Computable, in the sense that can be computed, but can also be set by the user.

Reply:
Right. Well, "an editable computed field" would be more descriptive to me in that case since I think that's the official term Odoo uses for such fields. But anyway, not that important.